### DISCUSSION TITLE
Help developers please try to fix mali gpu problems🥺🙏

### DISCUSSION THREAD

**Discussion by Anselshiu** [skyline-emu/skyline#1552] [Q&A] [answered]
😔I think many users use mali gpu,so really bad performance and even can't play,can devs fix it cuz this is quite a big problem🥲💔

**Comment by yhak22** [comment, marked as answer]
The dev is working hard for both snapdragon and mali user .
But for better performance the snapdragon is the choice.
Mali has problems in emulation apps and it's annoying but the devs are doing thier best.
